internal/kafka/internal/workers/clusters_mgr.go

```diff
@@ -842,6 +842,30 @@ func (c *ClusterManager) buildResourceSet() types.ResourceSet {
 		kasFleetshardNamespace = kasFleetshardQEAddonNamespace
 	}
 
+	// For standalone clusters, make sure that the namespaces is read from the config
+	// and that they are created before the pull secrets that references them
+	if cluster.ProviderType == api.ClusterProviderStandalone {
+		strimziNamespace = c.DataplaneClusterConfig.StrimziOperatorOLMConfig.Namespace
```

Comment:
Is there a need to check for empty string? (in case the namespace is not specified/created?)

Reply:
Created followup JIRA: https://issues.redhat.com/browse/MGDSTRM-8344